mj_models/include/mj_models/mj_mushr.hpp

```diff
@@ -1,15 +1,28 @@
 #pragma once
 
-#include "mj_models/MushrControl.h"
-#include "mj_models/MushrPlan.h"
 #include "mj_models/mj_copy.hpp"
 
+#include <mj_models/MushrControl.h>
+#include <mj_models/MushrPlan.h>
+#include <mj_models/MushrState.h>
+#include <mj_models/MushrFeedback.h>
+
 namespace mj_models
 {
 
 struct mushr_t
 {
   static constexpr std::size_t u_dim{ 2 };
+  struct sensors_t
```

Comment:
Maybe something more descriptive, like `ground_truth_pose_sensor`.

Reply:
I intended this to be for *any* sensor, not only pose (i.e. IMU)